src/main/java/org/ga4gh/drs/utils/cache/AccessCache.java

```diff
@@ -0,0 +1,34 @@
+package org.ga4gh.drs.utils.cache;
+
+import com.google.common.cache.CacheBuilder;
+import com.google.common.cache.CacheLoader;
+import com.google.common.cache.LoadingCache;
+
+public class AccessCache {
+
+    private LoadingCache<String, AccessCacheItem> cache;
+
+    public AccessCache() {
+        buildCache();
+    }
+
+    private void buildCache() {
+        cache = CacheBuilder.newBuilder()
+            .maximumSize(1000)
+            .build(
+                new CacheLoader<String, AccessCacheItem>(){
+                    public AccessCacheItem load(String key) {
+                        return new AccessCacheItem();
+                    }
+                }
+            );
+    }
+
+    public void put(String key, AccessCacheItem value) {
```

Comment:
Instead of taking the fully formed key as a string, I think it might be better for the put and get methods to take the object and access ids separately as parameters. Right now the scheme for mapping from object id and access id to key (object id : access id) is hardcoded in two places, but in the future if we want to change the scheme for example by adding a hash, it'd be better to have the cache take care of forming the key

Reply:
good point, I will update